relay-quotas/src/quota.rs

```diff
@@ -4,12 +4,12 @@ use std::str::FromStr;
 use serde::{Deserialize, Serialize};
 use smallvec::SmallVec;
 
-use relay_common::ProjectId;
+use relay_common::{ProjectId, ProjectKey};
 
 /// Data scoping information.
 ///
 /// This structure holds information of all scopes required for attributing an item to quotas.
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
since this is now Copy theoretically you can put it into ItemScoping without a lifetime

Reply:
Good point!

Reply:
Actually, I'll leave it like this. The `ItemScoping` type is quite large, and since one should only temporarily hold an item scoping, it's probably fine to keep it with a reference.